packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.ts

```diff
@@ -162,6 +167,32 @@ export class FlowsTableComponent implements OnInit {
     this.duplicateFlow$ = this.flowService.duplicate(flow.id);
   }
 
+  getTriggerIcon(flow: Flow) {
+    const trigger = flow.version.trigger;
+    switch (trigger.type) {
+      case TriggerType.WEBHOOK:
+        return 'assets/img/custom/triggers/instant.svg';
+      case TriggerType.PIECE: {
+        return 'assets/img/custom/triggers/periodic.svg';
+      }
+      case TriggerType.EMPTY:
+        throw new Error("Flow can't be published with empty trigger");
```

Comment:
no need to throw an error, just don't show anything by returning '';

Reply:
It case shouldn't happen, both are okay.